col/src/main/java/vct/col/ast/print/PVLPrinter.java

```diff
@@ -652,6 +660,17 @@ public void visit(Contract contract) {
         }
     }
 
+    private void printContractElement(ASTNode expr, String contractHead) {
```

Comment:
This one is also basic and generic, canyou put this in AbstractPrinter?

Reply:
Done.